.dotnet.azure/src/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -45,32 +63,12 @@ public string ApplicationId
     /// <summary>
     /// Initializes a new instance of <see cref="AzureOpenAIClientOptions"/>
     /// </summary>
-    /// <param name="version"> The service API version to use with the client. </param>
-    /// <exception cref="NotSupportedException"> The provided service API version is not supported. </exception>
-    public AzureOpenAIClientOptions(ServiceVersion version = LatestVersion)
+    public AzureOpenAIClientOptions()
         : base()
     {
-        Version = version switch
-        {
-            ServiceVersion.V2024_04_01_Preview => "2024-04-01-preview",
-            ServiceVersion.V2024_05_01_Preview => "2024-05-01-preview",
-            ServiceVersion.V2024_06_01 => "2024-06-01",
-            ServiceVersion.V2024_07_01_Preview => "2024-07-01-preview",
-            _ => throw new NotSupportedException()
-        };
         RetryPolicy = new RetryWithDelaysPolicy();
     }
```

Comment:
Is having a constructor for the ClientOptions with no service version a new pattern?

Reply:
For Azure libraries, I think it is -- for OpenAI, "no constructors with optional parameters" has been our target. Given this library isn't built on Azure.Core and is tied so directly to OpenAI, it feels like consistency with the latter is higher priority than consistency with the former.